packages/vite-plugin-svelte/src/utils/options.ts

```diff
@@ -288,6 +290,15 @@ function addSvelteKitOptions(options: ResolvedOptions) {
 	}
 }
 
+function handleDeprecatedOptions(options: ResolvedOptions) {
+	if ((options.experimental as any)?.prebundleSvelteLibraries) {
+		options.prebundleSvelteLibraries = (options.experimental as any)?.prebundleSvelteLibraries;
+		log.warn(
```

Comment:
This should be thrown as an error. Otherwise people update v-p-s and suddenly svelte dependencies are no longer prebundled and if they don't check warnings that might have scrolled by quickly, they'll miss it.